terraform/state_bucket/bucket.tf

```diff
@@ -0,0 +1,51 @@
+resource "aws_s3_bucket" "bucket" {
+  bucket = "osquery-foundation-terraform-state"
+  acl    = "private"
+  region = "eu-west-2" 
```

Comment:
Is the choice of EU intentional? I'd love to hear about why. (and possible we should drop that into the README)

Reply:
Just copy-paste if we're defaulting to `us-east-1` then that's probably best, ideally, we should keep all resources to a single region as it makes for better visibility and easier configuration of services such as guard-duty.

Reply:
Okay, not any kind of data privacy stuff then.

I think most of osquery development is US based, likely centered in CA. But I don't have any real strong feelings about where it should live

Reply:
`us-east-1` is fine